rmgpy/solver/base.pyx

```diff
@@ -746,20 +752,28 @@ cdef class ReactionSystem(DASx):
                             conversion = 1 - (y_core_species[index] / y0[index])
 
                     if invalid_objects == []:
-                        #species flux criterion
-                        if len(edge_species_rate_ratios) > 0:
-                            ind = np.argmax(edge_species_rate_ratios)
-                            obj = edge_species[ind]
-                            logging.info('At time {0:10.4e} s, species {1} at rate ratio {2} was added to model core '
-                                         'in model resurrection process'.format(self.t, obj,edge_species_rates[ind]))
-                            invalid_objects.append(obj)
-
-                        if total_div_accum_nums and len(total_div_accum_nums) > 0:  #if dynamics data available
-                            ind = np.argmax(total_div_accum_nums)
-                            obj = edge_reactions[ind]
-                            logging.info('At time {0:10.4e} s, Reaction {1} at dynamics number {2} was added to model core '
-                                         'in model resurrection process'.format(self.t, obj,total_div_accum_nums[ind]))
-                            invalid_objects.append(obj)
+                        if self.retry < self.max_retries:
+                            #species flux criterion
+                            if len(edge_species_rate_ratios) > 0:
+                                ind = np.argmax(edge_species_rate_ratios)
+                                obj = edge_species[ind]
+                                self.retry += 1
```

Comment:
Maybe we should count each retry every time we encounter a `DASPK` error, regardless whether we enter the 2 if blocks which increment the self.retry?
```
 logging.error("Trying to step from time {0} to {1} resulted in a solver (DASPK) error: "
                                  "{2!s}".format(prev_time, step_time, e))
if self.retry >= self.max_retries:
    raise RuntimeError
self.retry += 1
```
Not sure 🤔.  Also since it's series of 3 ifs and not if/elifs, I'm not sure if we would ever enter more than one of those.

Reply:
I changed it to just increment on DASPK errors because that makes more sense.